scripts/modeling_toolbox/utils.py

```diff
@@ -13,14 +13,10 @@ def load_data(data_uri, nrows=None):
     data_df.index = data_df.id
     lowercase = lambda x: str(x).lower()
     data_df.rename(lowercase, axis='columns', inplace=True)
-    data_df.drop(['dimension_x'], axis=1, inplace=True)
     data_df.rename(columns={'attack':'rendition',
-                            'title':'source',
-                            'dimension': 'dimension_x'},
+                            'title':'source'},
                    inplace=True)
     data_df['rendition'] = data_df['rendition'].apply(lambda x: set_rendition_name(x))
-    data_df['dimension_y'] = data_df['rendition'].apply(lambda x: int(x.replace('p','').split('_')[0]))
-    data_df['size_dimension_ratio'] = data_df['size'] / (data_df['dimension_y'] * data_df['dimension_x'])
```

Comment:
Why are we removing `size_dimension_ratio`?

Reply:
This code was for previous version of VideoAssetProcessor, which didn't calculate some meta-features.

Reply:
Ah I see and VideoAssetProcessor now [computes size_dimension_ratio](https://github.com/livepeer/verification-classifier/blob/8f48463dd6b1cd9def1ec19378562781af6e9e8e/scripts/asset_processor/video_asset_processor.py#L451) so that should already be included in the data read from a file right?